packages/react-jss/src/createHoc.js

```diff
@@ -121,23 +118,22 @@ export default function createHOC<
         this.unsubscribeId = themeListener.subscribe(this.context, this.setTheme)
       }
     }
-
+    // TODO: remove this lifecycle when migrating to new Context API
     componentWillReceiveProps(nextProps: OuterPropsType, nextContext: Context) {
       this.context = nextContext
```

Comment:
I'm not sure what this line does, maybe try removing it and run the tests to see if anything breaks.

Reply:
Hmm yeah its weird, I just commented it out and nothing breaks.